src/pages/pokemons-page/pokemons-page.component.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import Pokemon from 'Components/pokemon/pokemon.component';
+import SearchBox from 'Components/search-box/search-box.component';
+import Pagination from 'Components/pagination/pagination.component';
+
+import './pokemons-page.styles.scss';
+
+class PokemonsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchField: '',
+      paginationWindow: false,
+      togglePaginationWindow: () =>

```

Comment:
why do you store functions in the state of the component? you can make it just common class methods.

Reply:
Oh, truly